### PR TITLE
fix(MultipleTooltip): changed tooltips and it's backdrop styles

### PR DESCRIPTION
**Pull request by shvorak**
Hi

In this PR, I propose to change the tooltip styles

1. The background color of the tooltip itself has been changed with the help of the product design team.
2. Backdrop style is suggested to correct the wrong appearance in different themes.

As a basic idea, I took the background color variable and made a gradient.

| Before (AS IS) | After (TO BE) |
|--------|--------|
| image | image |
| image | image |

### CODE REVIEW

**Comment by shvorak** [comment]
UPD: I just received approval from the product design team for the 2nd point.

File: src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss [hunk 1]
```diff
@@ -2,26 +2,30 @@
 
 $block: '.#{variables.$ns}multiple-tooltip';
 
-.g-root_theme_dark #{$block},
```
Comment (Lunory) [inline src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss:5]: btw why did you change specificity? 🤔

File: src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss [hunk 2]
```diff
@@ -2,26 +2,30 @@
...
 }
 
+.g-root_theme_dark-hc {
+    --multiple-tooltip-item-bg-color: var(--g-color-private-white-150-solid);
+}
+
 .g-root_theme_light #{$block},
 .g-root_theme_light-hc #{$block} {
-    --multiple-tooltip-item-bg-color: #7a7a7a; // TODO: color variable will appear in uikit 5
+    --multiple-tooltip-item-bg-color: var(--g-color-private-black-550-solid);
```
Comment (Lunory) [inline src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss:23]: Should return `TODO`, using private color is outside rules

File: src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss (hunk 1 above)
Comment (shvorak) [reply src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss:5]: It's mistake

File: src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss (hunk 2 above)
Comment (shvorak) [reply src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss:23]: Do you have a proper solution for these cases?

Comment (DarkGenius) [reply src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss:23]: It should be noted that a non-private variable for this case is expected in the next major uikit

**Review by Lunory — APPROVED** [review]
